docs/API.md

```diff
@@ -1238,11 +1292,36 @@ A *batch id*.
 
 #### `"error"`
```

Comment:
Maybe you can add
````markdown
Trying to add a batch when a batch is still running for the current user
```json
{
  "jsonrpc": "2.0",
  "error": {
    "data": {
      "creation_time": "2021-09-27 07:33:40",
      "batch_id": 1
    },
    "code": -32603,
    "message": "Batch job still running"
  },
  "id": 1
}

```
````

Reply:
Good suggestion. Done.